src/Microsoft.Azure.WebJobs.Host/Listeners/ListenerFactoryContext.cs

```diff
@@ -39,6 +43,11 @@ public ListenerFactoryContext(FunctionDescriptor descriptor, ITriggeredFunctionE
         public ITriggeredFunctionExecutor Executor { get; private set; }
 
         /// <summary>
+        /// Gets the web jobs exception handler
+        /// </summary>
+        public IWebJobsExceptionHandler ExceptionHandler { get; private set; }
```

Comment:
Good question - this is encouraging an existing pattern which we see in blob listener, queue listener, and (now) service bus listener.  However, it all could be done via caching from the job host config - I'll do an iteration to see how complicated that path is.

Didn't realize that the core listeners / extensions used the config.Initialize call!